transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -188,7 +188,11 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("linux")) {
             throw new IllegalStateException("Only supported on Linux");
         }
-        NativeLibraryLoader.load("netty-transport-native-epoll", PlatformDependent.getClassLoader(Native.class));
+        String []libraryNames = new String[]{
+          "netty-transport-native-epoll",
+          "netty_transport_native_epoll"
```

Comment:
grepping through I don't see where the shared object gets its name.  Is it just derived from the artifact Id?  I wouldn't mind making the change in this PR, but I'm afraid I don't know maven that well.